src/controller/python/OpCredsBinding.cpp

```diff
@@ -369,11 +373,74 @@ void pychip_OnCommissioningStatusUpdate(chip::PeerId peerId, chip::Controller::C
     return sTestCommissioner.OnCommissioningStatusUpdate(peerId, stageCompleted, err);
 }
 
+/**
+ * Call of pychip_OpCreds_AllocateControllerFoCustomCommissioningFlow allows allocating a device commissioner that will perform
```

Comment:
nit: missing a `r` from the name of the method. Maybe don't put the name there at all as it is very long anyway. Just say `Allocates a controller that does not use auto-commisioning`.

Reply:
done